components/item-full.js

```diff
@@ -98,6 +112,47 @@ function ItemEmbed ({ item }) {
     )
   }
 
+  // if the link is to a rumble video, render the video
+  if (url.host === 'rumble.com') {
+    if (parts[0] === 'embed' && /^\w+$/.test(queryParams.pub)) {
+      return (
+        <div className={styles.youtubeContainerContainer}>
+          <div className={styles.youtubeContainer}>
+            <iframe
+              style={{ width: '100%', height: '100%' }}
+              title='Rumble Video'
+              allowFullScreen=''
+              src={item.url}
+            />
+          </div>
+        </div>
+      )
+    } else {
+      return null
```

Comment:
added these return nulls for now As i was hoping to create some logic for when the user's link does Not follow the expected embed format but at the moment no logic for that in rumble